Parsers/ASimProcessEvent/Parsers/vimProcessCreateTrendMicroVisionOne.yaml

```diff
@@ -0,0 +1,233 @@
+Parser:
+  Title: Process Create ASIM parser
+  Version: '0.1.0'
+  LastUpdated: Oct 10, 2023
+Product:
+  Name: Trend Micro Vision One
+Normalization:
+  Schema: ProcessEvent
+  Version: '0.1.0'
+References:
+- Title: ASIM Process Schema
+  Link: https://aka.ms/ASimProcessEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+- Title: Trend Micro Vision One documentation
+  Link:
+    https://docs.trendmicro.com/en-us/enterprise/trend-vision-one/xdr-part/search-app/data-mapping-intro/data-mapping-detecti.aspx
+    https://docs.trendmicro.com/en-us/enterprise/trend-vision-one/xdr-part/search-app/data-mapping-intro/data-mapping-endpoin.aspx
+    https://automation.trendmicro.com/xdr/api-v3#tag/Observed-Attack-Techniques-Pipeline/paths/~1v3.0~1oat~1dataPipelines~1%7Bid%7D~1packages~1%7BpackageId%7D/get
+    https://automation.trendmicro.com/xdr/api-v3#tag/Observed-Attack-Techniques/paths/~1v3.0~1oat~1detections/get
+Description: |
+  This ASIM parser supports normalizing Trend Micro Vision One logs to the ASIM Process Create normalized schema. Trend Micro Vision One events are captured through Trend Vision One data connector which ingests XDR logs into Microsoft Sentinel through the Trend Vision One API.
+ParserName: vimProcessCreateTrendMicroVisionOne
```

Comment:
@jayeshprajapaticrest @niralishah-crest 
You didn't had logs for Process Termination or the product doesn't provide for it?

Reply:
We don't have logs for Process Termination in Trend Micro Vision One.